bb-pilcom/README.md

```diff
@@ -0,0 +1,11 @@
+### BB PILCOM
+
+#### Using Optimized Relations
+
+The "use_optimized" feature flag is on by default. This means that generated files will include any optimized relations specified in BBFiles (specifically in `get_optimized_relations_file_names`);
+`cargo build --workspace --release`
```

Comment:
can you confirm this is what `bootstrap` in this dir would do?

Reply:
good shoutout, i'll double check before merging